.github/workflows/dapr-automerge.yml

```diff
@@ -0,0 +1,24 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation and Dapr Contributors.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+name: dapr-automerge
+
+on:
+  schedule:
+    - cron: '*/10 * * * *'
+  workflow_dispatch:
+jobs:
+  automerge:
+    name: Automerge and update PRs.
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: pip install PyGithub
+      - name: Automerge and update
+        env:
+          GITHUB_TOKEN: ${{ secrets.DAPR_BOT_TOKEN }}
```

Comment:
Can we scope this to just the step that needs it rather than the whole workflow?

Reply:
The env is already in a step.

Reply:
Right you are, my brain missed the narrow scoping entirely, nothing to see here, carry on.